src/applications/vaos/components/InfoAlert.jsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export default function InfoAlert({
+  backgroundOnly = false,
+  children,
+  headline,
+  level = 2,
+  status,
+}) {
+  const H = `h${level}`;
```

Comment:
Taken from the `AlertBox` component. Defaults to `h2` for accessibility reasons but the font size gets set to `h3`.